test/openwith.html

```diff
@@ -8,26 +8,23 @@
             .openwith {
                 margin-top: 50px;
             }
-
             .container {
                 display: flex;
                 height: 100%;
                 width: 100%;
                 align-items: center;
                 justify-content: center;
             }
-
             .container > div {
+                height: 100%;
```

Comment:
Mixing `height: 100%` with flex containers/children (especially in column mode) can cause subtle bugs in older browsers. Can this be accomplished using only flexbox?